### PR TITLE
[INFINITY-3173] Add a mount volume test for hello-world

### PR DESCRIPTION
**Pull request by gabrielhartmann**
This test ensures mount volumes on shared executors and on individual tasks work.  It is currently blocked by DCOS-21261.

### CODE REVIEW

**Comment by nickbp** [comment]
Will mount volumes always be added in test clusters, or should this test be gated on whether the test cluster was configured with mount volumes?

May still want the latter even if the former is the case -- e.g. CCM clusters probably won't be getting mount volumes.

**Comment by benclarkwood** [comment]
I would add a "mount_volume" mark to the test. Then a dev can easily disable it when doing manual testing.

**Comment by gabrielhartmann** [comment]
@nickbp @benclarkwood: Thanks guys.  What do you suggest @mellenburg ?  How will we know if a cluster has MOUNT volumes available or not?

**Comment by mellenburg** [comment]
@gabrielhartmann I actually don't have a good method for tests to check if volume mounts are enabled without SSHing into every node or making authenticated mesos requests. I would suggest disabling the test by default and then use some environment variable to enable it when you know the launch config will provide the desired volumes to support the test and then later implement the approach that asks mesos for its resources and skips based on that

**Comment by gabrielhartmann** [comment]
@nickbp @benclarkwood: I spoke with @mellenburg offline.  Every `hello-world` cluster will have MOUNT volumes.  We'll figure out selective enablement later.

File: frameworks/helloworld/tests/test_mount_volumes.py [hunk 1]
```diff
@@ -54,7 +102,7 @@ def test_kill_node():
 
 
 @pytest.mark.sanity
-def test_kill_agent():
+def test_kill_agent(mount_volumes):
```
Comment (gabrielhartmann) [inline frameworks/helloworld/tests/test_mount_volumes.py]: @mellenburg: Is adding this fixture to both tests going to add mount volumes twice?

File: frameworks/helloworld/tests/test_mount_volumes.py [hunk 2]
```diff
@@ -1,19 +1,67 @@
...
 from tests import config
 
+import dcos_launch
+import dcos_test_utils
+
 log = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope='session')
+def mount_volumes():
```
Comment (gabrielhartmann) [inline frameworks/helloworld/tests/test_mount_volumes.py]: Why is this added as a fixture in the test code?  Shouldn't it be a script called after cluster provisioning succeeds and run once to provide MOUNT volumes for the cluster?  Then tests can expect a cluster with MOUNT volumes.

File: frameworks/helloworld/tests/test_mount_volumes.py (hunk 1 above)
Comment (mellenburg) [reply frameworks/helloworld/tests/test_mount_volumes.py]: No, session scoped dependencies mean only once per run.

File: frameworks/helloworld/tests/test_mount_volumes.py (hunk 2 above)
Comment (mellenburg) [reply frameworks/helloworld/tests/test_mount_volumes.py]: Moved the code to `tools` and added a hook to run it in CI instead
